README.md

```diff
@@ -41,7 +41,7 @@ apt-get install nodejs
 apt-get install npm
 npm install -g bower
 ln -s /usr/bin/nodejs /usr/bin/node
-pip install django
+pip install https://www.djangoproject.com/download/1.7.8/tarball/
```

Comment:
I tried to install the whole thing yesterday, and it turns out that the default django installation of version 1.8 is incompatible with some of our existing codes. Though it is easy to fix the code to match newer version, I wonder should we keep the code compatible to the newest versions of our dependencies throughout the whole implementation process or should we stick to a particular version.